src/cursesdef.h

```diff
@@ -1,7 +1,7 @@
 #ifndef CURSESDEF_H
 #define CURSESDEF_H
 
-#if (defined TILES || defined _WIN32 || defined WINDOWS)
+#if (defined TILES || defined SDLTILES || defined _WIN32 || defined WINDOWS)
```

Comment:
ah yes, something went wrong so I manually tried to revert it, must have missed this one